website/docs/cdktf/concepts/resources.mdx

```diff
@@ -370,6 +370,84 @@ import imports.kubernetes.namespace.NamespaceMetadata;
 
 </CodeTabs>
 
+## Refactoring & Renaming Resources
+
+By default moving and/or renaming resource will be regarded with the intent to destroy the resource and re-create it. If such behavior must be avoided, but there is need to refactor your infrastructure definitions and/or rename resources in it, the function `moveTo` can serve your purpose.
```

Comment:
```suggestion
By default moving and/or renaming resources will be regarded with the intent to destroy the resource and re-create it. If such behavior must be avoided, but there is need to refactor your infrastructure definitions and/or rename resources in it, the function `moveTo` can serve your purpose.
```

Also: It seems like there's quite a bit of passive voice that I learned we usually try to avoid when writing docs.